client/src/components/userSettings/ChangePassword.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { ChangePasswordField } from './ChangePasswordField';
+import { Grid, Typography, TextField, Box, Button } from '@material-ui/core';
+import { userSetingsStyles } from './userSettings.style';
+import { USER_PASSWORD_MUTATION } from './queries/changeUserPassword';
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
+import Snackbar from '@material-ui/core/Snackbar';
+import { useMutation } from '@apollo/react-hooks';
+
+function Alert(props: AlertProps) {
```

Comment:
Please keep our convention by using const and arrow function instead of the "function" keyword. FIx it across all files.

Reply:
You can also move alert to other file